### ISSUE TITLE
Lomap data storage

### ISSUE THREAD

**Issue by gitblanc** [Arquisoft/lomapSpec#8]
I want to talk about the directory where the locations of each user will be stored.
I talked about it on https://github.com/Arquisoft/lomapSpec/issues/4 but nobody answered, so I create an issue for people that didn't read it.
In my case, I am creating a new folder called `justforfriends` (this name can be changed) where I store the locations file (in my case `locations.json`). So in this directory (stored on the root directory of the user's pod) i give permissions of editing to all my friends and ownership to myself.
I add pics of the structure:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/87705461/230877966-2693ec9a-96a1-4b3e-904f-c5c4f22c7c96.png">

<img width="801" alt="image" src="https://user-images.githubusercontent.com/87705461/230878163-03a47493-5f69-448d-924d-fec9d8ad2ba9.png">

PLEASE, tell me what you think, prefer and want to improve!

**Comment by aaronof17** [comment]
In my case I use the folder 'private', and here is where I store the locations file. I think that is not necesary to use other folder

**Comment by gitblanc** [comment]
> In my case I use the folder 'private', and here is where I store the locations file. I think that is not necesary to use other folder

Well, it's true that the folder `justforfriends` has the same permissions as private, but I think that separate directories is a better idea just for style of code. I saw the viade spec and they stored all the data on a directory called ´viade´.
In conclusion, I think it's better to separate the app storage and the pod storage.

**Comment by Omitg24** [comment]
I agree with @gitblanc, we should create a separate directory for our PODs inside of private folder, so we do not overload our PODs

**Comment by gitblanc** [comment]
Do you mind to call the directory `lomap` in the root directory? Or do you prefer another name?

**Comment by Omitg24** [comment]
I'm okay with that name for the directory.
